components/atoms/ActionButton.js

```diff
@@ -40,7 +40,9 @@ export function ActionButton(props) {
         }
         ${props.secondary && !props.disabled ? secondaryStyle : props.className}
         ${props.custom && !props.secondary ? props.custom : ""}
-        ${props.disabled ? disabledStyle : props.className} link`}
+        ${props.disabled ? disabledStyle : props.className}
+        ${props.className.indexOf("w-") !== -1 ? null : "w-max"}
```

Comment:
@Moro-Code I put it there because the link goes full screen without the w-max but when we put the w-max we can't change the width it in the props.className later. I'm not sure if it's the best way to approach it.